app/views/budgets/ballot/_progress_bar.html.erb

```diff
@@ -26,4 +36,6 @@
       </span>
     </p>
   </span>
-</div>
+  </div>
+<% end %>
+
```

Comment:
Could you elaborate please... do you mean that line 41 isn't a newline character? or do you mean to delete the line so that it becomes more like:

```
  </div>$
<% end %>$
```

Reply:
Right, the last line should not be empty (so there should be only 40 lines in this case) and the last line (line 40 in this case) should have a newline character at the end :+1:.